setup/quickstart/halyard-gce.md

```diff
@@ -1,13 +1,15 @@
 ---
 
 layout: single
-title:  "Halyard on GCE Quickstart"
+title:  "Try out Halyard on GCE"
 sidebar:
   nav: setup
 ---
 
 {% include toc %}
 
+Note: we recommend that you install Spinnaker following the [standard setup directions](/setup/) rather than the guides listed here.
```

Comment:
I don't know if "guides," plural, works well here. Because now they're not listed. Also, it might be good to say what this is for if it's not recommended. 

"We recommend you install...rather this using this guide, which is just a set of commands to get your Spinnaker up and running on GCS and GCE."

I think what we might decide to do is delete all these and redirect to /setup/install/.  @demobox WDYT?

Also, @nhayes you can indent this as a "note" using `> ` at the start of each line.